configure.ac

```diff
@@ -185,6 +177,23 @@ AS_CASE([$host],
 )
 AC_SUBST([ARCH_ABI])
 
+AC_ARG_ENABLE([host-optimization],
+	      [AS_HELP_STRING([--disable-host-optimization],
```

Comment:
What's the use case (other than testing) for this option? @muvarov's suggestion to cross compile for an otherwise-unsupported ARCH like Xtensa seems preferable.